src/main/groovy/de/dkfz/roddy/execution/jobs/BEJob.groovy

```diff
@@ -107,50 +108,42 @@ class BEJob<J extends BEJob, JR extends BEJobResult> implements Comparable<BEJob
 
     BatchEuphoriaJobManager jobManager
 
-    BEJob(BEJobID jobID,
-          String jobName,
-          File tool,
-          String toolScript,
-          String toolMD5,
-          ResourceSet resourceSet,
-          Collection<BEJob> parentJobs,
-          Map<String, String> parameters,
-          BatchEuphoriaJobManager jobManager,
-          JobLog jobLog,
-          File workingDirectory,
-          String accountingName = null) {
-        this.jobID = Optional.ofNullable(jobID).orElse(new BEJobID())
-        this.jobName = jobName
+    BEJob(BEJobID jobID,                        // can be null for FakeBEJob
```

Comment:
Not sure if needed, but you could mention in the readme that this is changed since it breaks the API